pkg/version/version.go

```diff
@@ -34,26 +30,31 @@ type CiliumVersion struct {
 	GoRuntimeVersion string
 	// Arch is the architecture where Cilium was compiled
 	Arch string
+	// AuthorDate is the git author time reference stored as string ISO 8601 formatted
+	AuthorDate string
 }
 
+// Version is set during build
 var Version string
 
-func versionFrom(versionString string) CiliumVersion {
-	cver := CiliumVersion{}
-	output := strings.Replace(versionString, "Cilium ", "", 1)
-
-	fields := strings.Split(output, " ")
-	cver.Version = fields[0]
-	cver.Revision = fields[1]
-	cver.Arch = fields[len(fields)-1]
-	cver.GoRuntimeVersion = runtime.Version()
+// FromString converts a version string into struct
+func FromString(versionString string) CiliumVersion {
```

Comment:
There's no need for this function to be public, right?

Reply:
it is used [here](https://github.com/stevenceuppens/cilium/blob/34529cf442cd48298c2c63799592b2d020b200d5/cilium/cmd/version.go#L82), outside of this package

Reply:
My bad, the files have the same name so that's why I miss it.